botocore/compat.py

```diff
@@ -146,8 +146,12 @@ def ensure_bytes(s, encoding='utf-8', errors='strict'):
 from collections import OrderedDict
 
 
-import xml.etree.cElementTree
-XMLParseError = xml.etree.cElementTree.ParseError
+try:
+    import xml.etree.cElementTree as ETree
+except ImportError:
+    # cElemenTree does not exist from Python3.9+
```

Comment:
```suggestion
    # cElementTree does not exist from Python3.9+
```

Reply:
Thanks, feels good to be reminded of all my drunk-commits lol. Fixed the typo.